docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -100,31 +143,68 @@ The database settings screen will give you a number of options for your applicat
 
 Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
 
-### Additional Options
+#### Configuring RDS for Metabase OUTSIDE ElasticBeanstalk (recommended)
+
+In AWS Console, enter RDS in the search box or select the service from the dropdown button on the top left of the page.
+
+Once inside RDS, click on the `Create database` button
+![RDS Create Database](images/RDSCreateDatabase.png)
+
+On the Create Database menu, you have to select MySQL or PostgreSQL as engine types, as these two are the ones that Metabase support as the Application Database (where Metabase will save all of its configurations). For this example we will choose PostgreSQL on its latest version available in AWS at this time (12.4-R1)
```

Comment:
```suggestion
On the Create Database menu, under **Chose a database creation method**, select "Standard create". Under **Engine options**, select "PostgreSQL." This application database will store all of Metabase's configurations, dashboards, questions, etc. 
```

Do we need to mention MySQL?

Reply:
I changed this part as well in the latest commit and made bullet points for each section